File: Kody-master/klasa 2/python/wejscie-wyjscie.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  wejscie-wyjscie.py
# duck typing

def main(args):
    a = int(input("Podaj pierwsza liczbe: "))
    print(a)
    
    b = int(input("Podaj druga liczbe: "))
    print(b)
    
    print("Suma: ", a + b)
    print("Różnica: ", a - b)
    print("Iloczyn: ", a * b)
    print("Iloraz: ", a / b)
    
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: klasa 2/cpp/wejscie-wyjscie.cpp
#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
	int a, b;
    a = b = 0;
    
    cout << "Podaj pierwsza liczbe: ";
    cin >> a;
    cout << a;
      
    cout << endl << "Podaj druga liczbe: ";
    cin >> b;
    cout << b;
    
    cout << endl << "Suma: " << a + b << endl;
    cout << endl << "Różnica: " << a - b << endl;
    cout << endl << "Iloraz: " << a / b << endl;
    cout << endl << "Iloczyn: " << a * b << endl;
    
	return 0;
}
